### PR TITLE
Universal

### PR DESCRIPTION
**Pull request by kkarayannis**

### CODE REVIEW

**Comment by kkarayannis** [comment]
##Areas to test:
* iPad
* Landscape orientation
* Product Categories and Product Types views
* Poster view
* Order Review (Square, Polaroid, Circle)
* iOS 7

File: Kite SDK/PSPrintSDK/ViewController.m [hunk 1]
```diff
@@ -35,6 +38,55 @@ -(void)viewDidAppear:(BOOL)animated{
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+#ifndef DEBUG
+    self.navigationController.navigationBar.hidden = YES;
```
Comment (dbotha) [inline Kite SDK/PSPrintSDK/ViewController.m]: I get why you're putting this in here but I wonder if it's the best idea. It kind of makes the sample app difficult to follow if you're coming to it as an integrating developer. Thoughts?

Comment (kkarayannis) [inline Kite SDK/PSPrintSDK/ViewController.m]: You are right. I'll make another app to use for App Store submission to keep this clean(er)

Comment (dbotha) [inline Kite SDK/PSPrintSDK/ViewController.m]: Cool, I'd also move to using the generic iOS Developer / iOS Distribution identities for Code Signing Identity so anyone coming along can build either straight away with no modifications. I think currently release is a Kite Tech LTD specific distribution profile.

**Pull request merged**